Magazine Exercise Victor

vicc30 opened the pull request:
Day 1 Exercise. Victor Cruz

vicc30 committed:
initial commit

vicc30 commented:
## Comments about my exercise.

* This is my exercise. 
* I am going to make the changes with what we learned from yesterday and I will do the responsive part.
* I may update my magazine to another more challenging.

![captura-pantalla-desktop](https://user-images.githubusercontent.com/61885157/121594287-79f86780-ca02-11eb-833c-fd308af41d91.png)

vicc30 committed:
Magazine Victor v2
Added new magazine with the  suggested changes.

vicc30 commented:
## Exercise day 2 - second commit

* Added a new magazine template. [Original magazine - view](https://files.magzter.com/resize/magazine/1402416838/1602696290/view/3.jpg)
* Added BEM style tags
* Added responsive design with media queries.
* TODO: update media queries. The media queries are not working as desired. (Resolved commit - 3)

![pantalla-hw2](https://user-images.githubusercontent.com/61885157/121602844-80d8a780-ca0d-11eb-8f1d-9ccf0ea2f28a.png)

vicc30 committed:
Fixed: Display view
Fixed display view on all screens.

vicc30 committed:
Code update
Update according to suggestions

vicc30 commented:
## Commit 4 - Update Code.

Code updated according to suggestions.

 * Added a class with a descriptive name to elements that had a rule. Mostly p tags.
 * Removed %  to font-size rule in the root. Used px instead in the root to determine rem in the body.
 * The numbering of the headers is respected. h1, h2, etc.
 * W3 checked HTML and CSS. No errors.
 
I leave the rule `width: 30%` inside  `.magazine__leftTop__new--borderTopGray::before`  because this width is used to determine the size of the border according to container.

vicc30 committed:
Added Script to find HTML tags

vicc30 committed:
update variables tags

vicc30 committed:
Make js as function
Receives a string (HTML) and returns an object with tags and text.

vicc30 committed:
Update function to show nested  tags

vicc30 committed:
added correct return

crisostomoglobant commented on app.js:
```diff
@@ -0,0 +1,230 @@
+    firma: "Sin firma"
+  };
+
+  function getMainTitle(html, obj) {
```
en cada function dentro de getTags tienes acceso a `toReturn ` y `doc`  variables
quiere decir que no es necesario que se la pases a cada uno de ellas como lo hiciste en 

`function getMainTitle(html, obj) { ---> function getMainTitle() {`

```
  function getSubtitle() {
    const subtitle = doc
      .getElementsByTagName("h1")[0]
      .parentNode.getElementsByTagName("p").innerText;
      
    toReturn.subTitulo = subtitle === undefined ? "Sin subtitulo" : subtitle;
  }

```

vicc30 replied:
Ya quedó resuelto. Muchas gracias!

crisostomoglobant commented on app.js:
```diff
@@ -0,0 +1,230 @@
+    const title = html.getElementsByTagName("h1")[0].innerText;
+    obj.tituloPrincipal = title;
+  }
+
```
todo el naming debe estar en ingles

vicc30 committed:
Applied requested changes
Replaced all tags with English names.
Removed unnecessary inputs for call inner functions.
